crates/cli/src/main.rs

```diff
@@ -275,6 +279,69 @@ mod linux {
     }
 }
 
+#[cfg(target_os = "linux")]
+mod flatpak {
+    use std::ffi::OsString;
+    use std::path::PathBuf;
+    use std::process::Command;
+    use std::{env, process};
+
+    /// Restarts outside of the sandbox if currently running within it
+    pub fn try_restart_to_host() {
+        if let Some(bin_dir) = get_flatpak_bin_dir() {
+            let mut args = vec!["/usr/bin/flatpak-spawn".into(), "--host".into()];
```

Comment:
Seems like xdg's startup notify is broken because the pid changes when flatpak-spawn is called. The symptom seems to be that the app icon may not appear for a while (seems like the DE is rechecking every once in a while and eventual finds it. Looking into how this can be avoided.

Reply:
nvm, starting with --foreground fixes it.